docs/pages/foundations/messaging/priority_and_placement.js

```diff
@@ -0,0 +1,326 @@
+            productivity and inspiration.
+          </Text>
+
+          <Heading size="300" accessibilityLevel={4}>
+            Current components: Modal
+          </Heading>
+          <Text>
+            Blocking content slows a person down from their usual flow, so here are some guidelines
+            on when it’s okay to block content.
+          </Text>
+          <Box>
+            <Flex alignItems="center">
+              <Box padding={4} width="50%">
+                <Flex gap={1} alignItems="center">
+                  <Status type="ok" />
+                  <Text weight="bold">When to use</Text>
+                </Flex>
+              </Box>
+              <Box padding={4} width="50%" display="none" lgDisplay="block">
+                <Text weight="bold">Example wireframe</Text>
+              </Box>
+            </Flex>
+            <Box color="successWeak">
+              <Flex direction="column">
+                <Box color="successBase" height={4} width="100%" />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      To confirm that someone wants to peform a destructive/irreversible action
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <DeleteBoard />
+                  </Box>
+                </Box>
+                <Divider />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      When an issue is so severe that it’s hard to continue with a good experience
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <WebsiteBlocked />
+                  </Box>
+                </Box>
+                <Divider />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      To warn a person of something that can have serious consequences before they
+                      continue
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <SiteSpam />
+                  </Box>
+                </Box>
+                <Divider />
+                <Box display="flex" alignItems="center" direction="column" lgDirection="row">
+                  <Box padding={4} width="50%">
+                    <Text>
+                      As an interstitial that leads to a much better experience we think a person
+                      shouldn’t miss
+                    </Text>
+                  </Box>
+                  <Box padding={4} width="50%">
+                    <ExperienceTrends />
```

Comment:
Nit: But the "Delete" Button here seems wrong, maybe "Ok" instead? Hopefully not too hard to change....